server/src/Models/Game.ts

```diff
@@ -31,7 +31,7 @@ export class GameModel {
   };
 
   createGame = async (
-    userId: string,
+    userId: number,
```

Comment:
it stinks that anything taken off req is a string it seems have to double check but now that im taking this off the parsed user access token it can go back to `number`